pallets/schemas/src/migration.rs

```diff
@@ -85,27 +67,44 @@ pub mod v1 {
 // ==============================================
 /// Schema migration to v1 for pallet-stateful-storage
 /// This struct derives OnRuntimeUpgrade trait which is used to run the migration (check runtime)
-pub struct SchemaMigration<T: Config>(PhantomData<T>);
-
-impl<T: Config> OnRuntimeUpgrade for SchemaMigration<T> {
-	#[cfg(feature = "try-runtime")]
```

Comment:
What's the motivation for removing all this try-runtime code?  It's kind of out of scope for this PR.

Reply:
It was showing up as not tested and I was under the impression (wrong?) that it was unneeded. So trying to standardize the migration

Reply:
Yep all good, not sure we even need to keep migration logic anymore, since it must have already migrated all schemas on mainnet